toolkit/tools/imagegen/configuration/systemconfig.go

```diff
@@ -61,12 +73,24 @@ func (s *SystemConfig) IsValid() (err error) {
 				return fmt.Errorf("empty kernel entry found in the [KernelOptions] field (%s); remember that kernels are FORBIDDEN from appearing in any of the [PackageLists]", name)
 			}
 		}
+	}
+
+	// Validate the partitions this system config will be including
+	mountPointUsed := make(map[string]bool)
+	for _, partitionSetting := range s.PartitionSettings {
+		if err = partitionSetting.IsValid(); err != nil {
+			return fmt.Errorf("invalid [PartitionSettings]: %w", err)
+		}
+		if mountPointUsed[partitionSetting.MountPoint] {
+			return fmt.Errorf("invalid [PartitionSettings]: duplicate mount point found at '%s'", partitionSetting.MountPoint)
+		}
+		mountPointUsed[partitionSetting.MountPoint] = true
+	}
 
-		// for _, partitionSetting := range sysConfig.PartitionSettings {
-		// 	if err = partitionSetting.IsValid(); err != nil {
-		// 		return fmt.Errorf("invalid [PartitionSettings]: %w", err)
-		// 	}
-		// }
+	if s.Encryption.Enable {
+		if !mountPointUsed["/"] {
+			return fmt.Errorf("invalid [ReadOnlyVerityRoot] or [Encryption]: must have a partition mounted at '/'")
```

Comment:
>ReadOnlyVerityRoot [](start = 31, length = 18)

Similar comment here: is the error message still valid when we're not checking for `s.ReadOnlyVerityRoot.Enable`?